ang/civicase/activity/filters/directives/activity-filters.directive.js

```diff
@@ -32,7 +32,7 @@
      * @param {object} element element
      */
     function activityFiltersLink ($scope, element) {
-      var ts = $scope.ts = CRM.ts('civicase');
+      var ts = $scope.ts;
```

Comment:
use the `ts` service instead of getting the one in the scope:

https://github.com/compucorp/uk.co.compucorp.civicase/blob/master/ang/civicase-base/services/ts.service.js#L5